README.md

```diff
@@ -23,6 +23,9 @@ You will need the following things properly installed on your computer.
 
 * `ember serve`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
+* App can be run in two modes: 
```

Comment:
* change `npm install` na `yarn`
* surround every bash command with back ticks i.e. `ember s -e rails`
* add info that you can filter tests with `-f` 
* update information about deploy
* remove section further reading